engine/tests/core/graph/test_token.py

```
@@ -0,0 +1,132 @@
+    Weapon(label="sword", damage=10)
+
+    token = Token[Weapon](label="sword", durability=85)
+
+    # Instance var (on token)
+    assert token.durability == 85
+
+    # Delegates to base
+    assert token.damage == 10
+
+    # Update base affects token
+    base = Weapon.get_instance("sword")
```

Ruff (0.14.10)

Local variable `base` is assigned to but never used

Remove assignment to unused variable `base`

(F841)